kent2cky/GeoSearch
# #3 Add temperature converter

**What does the PR do?**

Add temperature converter

**Description of Tasks to be completed**

Have the page be able to convert temperature from fahrenheit to celcius and vice-versa.

**How should this be manually tested?**

- After cloning the repo, cd into it and run npm install
- After that, run the command gulp and it should be able to open up a page on your web browser.
- Go to the console tab of your browser's develop tools
- The result of a default conversion should be displayed on the console.
- You can also test it yourself by calling the functions on the` main.js` file and have the results displayed on the browser console.

**Any background context you want to provide?**

The temperature conversion functionality was implemented using the JavaScript object oriented pattern.

**What are the relevant pivotal tracker stories?**

#3
